src/main/java/org/jabref/gui/undo/UndoableRemoveEntries.java

```diff
@@ -0,0 +1,87 @@
+package org.jabref.gui.undo;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.event.EntriesEventSource;
+import org.jabref.model.strings.StringUtil;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class represents the removal of entries. The constructor needs
+ * references to the database, the entries, and the map of open entry editors.
+ * TODO is this map still being used?
```

Comment:
Can this TODO be resoled?

Reply:
It would appear the map is not being used, as there is no reference to entry editors in the class. Am I mistaken?